src/main/java/io/github/oliviercailloux/y2018/apartments/utils/JsonConvert.java

```diff
@@ -73,13 +72,9 @@ public static void apartmentToJson(Apartment a) throws IOException {
 	 */
 	public static void apartmentToJson(Apartment a, Path jsonPath) throws IOException {
 		Jsonb jsonb = JsonbBuilder.create();
-
-		try (BufferedWriter writer = Files.newBufferedWriter(jsonPath, StandardCharsets.UTF_8)) {
-			writer.write(jsonb.toJson(a));
-			writer.close();
-
-			LOGGER.info("Apartment have been converted with success");
-		}
+		Files.writeString(jsonPath,jsonb.toJson(a),StandardCharsets.UTF_8);
```

Comment:
Allez au plus simple SVP : adoptez la version à un seul argument. (OPT)

Reply:
Fix in commit 12d1c12213381bc05972b0cc51c70be71f83df47